external/build-external.js

```diff
@@ -0,0 +1,76 @@
+/**
+ * @fileoverview Builds all external sources. This clears the data/ folder here
+ * and allows all scripts in build/ to write new files there.
+ *
+ * This is intended for use by Cloud Build, or by site devs doing local work.
+ */
+
+// nb. This is important; we load dotenv here and pass environment down.
+// It's safe to call this inside the build scripts too, but we run them with
+// cwd=data/, so it can't find the env.
+require('dotenv').config();
+
+const fs = require('fs');
+const path = require('path');
+const glob = require('glob');
+const childProcess = require('child_process');
+const crypto = require('crypto');
+
+async function run() {
```

Comment:
I don't think I see any await, couldn't this just be script.... Like, nothing in the run function, i.e. no run function?

Reply:
You're right, but this matches the other files, and it makes it easier if it ever needs to do async work. What do you think?